### ISSUE TITLE
Fix undo and redo aria-label

### ISSUE THREAD

**Issue by sahalsaad** [accordproject/markdown-editor#164]
Currently both undo & redo aria-label shown as `bold`.
It should be `redo` & `undo`

**Comment by joeheyming** [comment]
I can take this

**Comment by joeheyming** [comment]
https://github.com/accordproject/markdown-editor/pull/166

**Comment by joeheyming** [comment]
If you create a separate git issue for eslint, I can try to update it.  It seems not to work for me.
